01_network_sysapi/headers/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_library(Headers STATIC)
+
+target_include_directories(Headers 
+	PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}"
+)
+
+target_sources(Headers
+	PUBLIC
+		header.hpp
+	PRIVATE
+		header.cpp
+)
+
+target_include_directories(Headers
```

Comment:
You don't have to include headers of Network library if it already includes own headers. CMake will do it transitively.